android/lint.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <!-- list of issues to configure -->
```

Comment:
add a link to http://tools.android.com/tips/lint-checks so people can quickly reference the issues